### PR TITLE
[ft-166147495]create-a-loan-challenge-3

### PR DESCRIPTION
**Pull request by ngireric123**
#### What does this PR do?
create a loan application
#### Description of Task to be completed?
have the following endpoint working
POST /api/v1/loans
#### How should this be manually tested?
I test this using postman
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
166147495
#### Screenshots (if appropriate)
Screenshot from 2019-05-21 09-10-09

#### Questions:

### CODE REVIEW

File: models/loan.js [hunk 1]
```diff
@@ -1,21 +1,48 @@
...
+   const res = await pool.query('SELECT * FROM loans WHERE email = $1 AND repaid = false', [email.trim()]);
+   if (res.rowCount < 1) {
+     return true;
+   }
+    return false;
+ }
 }
 
-export default loans;
+export default new loans;
```
Comment (Duncanian) [inline models/loan.js:48]: Why are you sing the new keyword here?

**Review by Duncanian — CHANGES_REQUESTED** [review]

File: controllers/loanControllers.js [hunk 2]
```diff
@@ -1,142 +1,40 @@
 import Joi from 'joi';
```
Comment (Duncanian) [inline controllers/loanControllers.js]: You still have joi here in your controller.

**Review by Duncanian — CHANGES_REQUESTED** [review]

File: controllers/loanControllers.js (hunk 2 above)
Comment (ngireric123) [reply controllers/loanControllers.js]: let me remove it also

File: middleware/isAdmin.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
-const adminAccess = (req, res, next) => {
+class admin{
+const adminAccess (req, res, next) => {
```
Comment (Duncanian) [inline middleware/isAdmin.js]: Why have you used the const keyword for your class function? Also give the function some spacing from the margin.

File: middleware/auth.js [hunk 4]
```diff
@@ -2,8 +2,8 @@ import jwt from 'jsonwebtoken';
 import dotenv from 'dotenv';
 
 dotenv.config();
-
-const isAuth = (req, res, next) => {
+class Authantication{
+static async isAuth  (req, res, next) => {
```
Comment (Duncanian) [inline middleware/auth.js]: Why are you using async here?

**Review by Duncanian — CHANGES_REQUESTED** [review]

File: middleware/auth.js [hunk 5]
```diff
@@ -2,8 +2,8 @@ import jwt from 'jsonwebtoken';
 import dotenv from 'dotenv';
 
 dotenv.config();
-
-const isAuth = (req, res, next) => {
+class Authantication{
```
Comment (Duncanian) [inline middleware/auth.js]: The authentication class is misspelled

File: middleware/auth.js [hunk 6]
```diff
@@ -2,8 +2,8 @@ import jwt from 'jsonwebtoken';
...
+static isAuth  (req, res, next) => {
```
Comment (Duncanian) [inline middleware/auth.js]: You have also not added some padding in your functions.

**Review by Duncanian — CHANGES_REQUESTED** [review]

**Review by Duncanian — APPROVED** [review]

**Pull request merged**